smdebug/pytorch/utils.py

```diff
@@ -79,5 +79,14 @@ def is_pt_1_8():
     return version.parse("1.8.0") <= PT_VERSION < version.parse("1.9.0")
 
 
+@lru_cache(maxsize=1)
+def is_pt_1_9():
+    """
+    Determine whether the version of torch is 1.8.x
+    :return: bool
+    """
+    return version.parse("1.9.0") <= PT_VERSION < version.parse("2.0.0")
```

Comment:
nit: version numbers can technically go over 9, so we should check instead that:
```
return version.parse("1.5.0") <= PT_VERSION < version.parse("1.10.0")
```

Reply:
It can go either way. In any case we would have to update this line when a new version comes up